coala_quickstart/interaction/Info.py

```diff
@@ -0,0 +1,15 @@
+def print_info(printer, info, colored=True):
+    """
+    Prints the given string.
+
+    :param printer:
+        A ConsolePrinter object used for console interactions.
+    :param info:
+        String to be printed.
+    :param colored:
+        Whether to use color in output or not.
+    """
+    if colored:
+        printer.print(info, color="blue")
+    else:
+        printer.print(info)
```

Comment:
Actually this is unnecessary - this is an artifact from a previous approach I used. Removed the whole function.